public/js/custom.js

```diff
@@ -4,3 +4,15 @@ $(".like-button").on('click', function() {
     $(".like-button").text('LIKE COUNT: ' + JSON.parse(data).likes);
   });
 });
+
+$(".in-application-search input").on('keyup', function() {
+  var query = $(this).val();
+  $.get('/api/search/' + query, function(data) {
+    $(".search-results").html('');
```

Comment:
it changes the html content of $(".search-results"). In other words the code above resets all the children elements of .search-results, it replaces them with '' empty string